jsonschema2pojo-gradle-plugin/src/main/groovy/org/jsonschema2pojo/gradle/GenerateJsonSchemaAndroidTask.groovy

```diff
@@ -57,4 +59,12 @@ class GenerateJsonSchemaAndroidTask extends SourceTask {
 
     Jsonschema2Pojo.generate(configuration, new GradleRuleLogger(logger))
   }
+
+  void setTargetVersion(JsonSchemaExtension configuration) {
+    if (!configuration.targetVersion) {
+      def compileJavaTask = project.getTasksByName("compileJava", false).first()
+      configuration.targetVersion = compileJavaTask.getProperties().get("sourceCompatibility")
+      logger.warn 'Using Gradle targetCompatibility as targetVersion for jsonschema2pojo: ' + configuration.targetVersion
```

Comment:
@joelittlejohn you have now merged this with the inconsistent log message, it should mention `sourceCompatibility` here, no?

same problem in `GenerateJsonSchemaJavaTask.groovy`

Reply:
Oops, thanks I will fix